### PR TITLE
USHIFT-337: Support topolvm disk, building external etcd, etc

### PR DESCRIPTION
**Pull request by sjug**
- Support topolvm disk, building external etcd, and alternative microshift repos.
- Add pidstat sample interval and additional RHEL repo.

**Which issue(s) this PR addresses**: https://issues.redhat.com/browse/USHIFT-337

### CODE REVIEW

File: ansible/roles/install-microshift/defaults/main.yml [hunk 1]
```diff
@@ -55,3 +62,7 @@ du_dirs:
   - /var/lib/containers/storage/
   - /usr/
   - /usr/bin/*
+
+lvm_disk: /dev/xvdb
```
Comment (ggiguash) [inline ansible/roles/install-microshift/defaults/main.yml:66]: This is the device name on AWS cloud, right? However, these are different in other environment, specifically in VMs created by libvirt.

File: ansible/roles/install-microshift/tasks/main.yml [hunk 2]
```diff
@@ -7,6 +7,11 @@
     state: present
     update_cache: true
 
+- name: create a volume group on top of secondary disk for topolvm
+  community.general.lvg:
+    vg: rhel
+    pvs: "{{ lvm_disk }}"
```
Comment (ggiguash) [inline ansible/roles/install-microshift/tasks/main.yml:13]: Can we make this conditional on existence of "rhel" vg? 
sudo vgdisplay -As

**Review by ggiguash — CHANGES_REQUESTED** [review]

**Comment by sjug** [comment]
Need to remove the configmap loading task and/or make it conditional.

**Pull request merged**